src/documents/consumer.py

```diff
@@ -191,6 +200,9 @@ def _ocr(self, pngs, lang):
         Performs a single OCR attempt.
         """
 
+        if not pngs:
```

Comment:
I'm slicing the PNG-list in `_get_ocr` into `headlist - middle - taillist`. For documents of lengths 3 and up, both the `headlist` and the `taillist` will be populated, for length 2 the `headlist` will be empty and for length 1 both the `headlist` and the `taillist` will be empty.

I moved the check into here to not clutter the `_get_ocr` implementation with checks for empty lists all over the place but rather ignore it in a central place.

What we could and probably should do though is add a check into the top of `_get_ocr` and fail if the initial list we get is empty. What do you think?